cic/BoardConfig.mk

```diff
@@ -20,8 +20,15 @@ BOARD_RECOVERYIMAGE_PARTITION_SIZE ?= 31457280
 BOARD_CACHEIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_CACHEIMAGE_PARTITION_SIZE ?= 104857600
 ##############################################################
+# Source: device/intel/mixins/groups/avb/true/BoardConfig.mk
+##############################################################
+BOARD_AVB_ENABLE := true
```

Comment:
Do we try to keep flags inside Kernelflinger? Is it possible in this case?

Reply:
No.  The BOARD_AVB_ENABLE flag is defined by Android, not kernelflinger. It is used to control to generate vbmeta image and sign images by AOSP code.  So should not put into kernelflinger.